### PR TITLE
chore: New GH workflow to validate VIB pipeline files

### PR DESCRIPTION
**Pull request by juan131**
### Description of the change

This PR adds a new GH workflow to validate VIB pipeline files by ensuring the **_verify_** phases are identical on both `vib-verify.json` & `vib-publish.json`. As it's explained in [testing guidelines](https://github.com/bitnami/charts/blob/main/TESTING.md#vib-verifyjson-vs-vib-publishjson):

> _it was decided that the verification process should be identical in both cases_

### Benefits

Ensure no discrepancies are introduced by mistake.

### Possible drawbacks

None

### Applicable issues

N/A

### Additional information

N/A

### Checklist

- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .github/workflows/vib-files-validator.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        run: |
+          exit_code=0
+          for dir in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            diff <(jq .phases.verify < "${dir}/vib-publish.json") <(jq .phases.verify < "${dir}/vib-verify.json") || exit_code=$?
+          done
+          if [[ $exit_code -ne 0 ]]; then
+            echo "::error:: Please ensure verify phase is aligned on VIB pipeline files"
+            exit "$exit_code"
+          fi
+        shell: bash
```
Comment (fmulero) [inline .github/workflows/vib-files-validator.yml:41]: I think setting the `shell` above `run` will be clearer

File: .github/workflows/vib-files-validator.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      - name: Get changed VIB pipeline files
+        id: changed-files
+        uses: tj-actions/changed-files@v37
+        with:
+          dir_names: "true"
+          dir_names_max_depth: "2"
+          files: |
+            .vib/**
+      - name: Validate VIB pipeline files
+        if: steps.changed-files.outputs.any_changed == 'true'
```
Comment (fmulero) [inline .github/workflows/vib-files-validator.yml:31]: We usually write one line `if`s inside of `${{``}}´ 
```suggestion
        if: ${{ steps.changed-files.outputs.any_changed == 'true' }}
```

**Review by fmulero — COMMENTED** [review]
Thanks a lot @juan131, good stuff

I left little comments, nothing important, could you review them?

File: .github/workflows/vib-files-validator.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        run: |
+          exit_code=0
+          for dir in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            diff <(jq .phases.verify < "${dir}/vib-publish.json") <(jq .phases.verify < "${dir}/vib-verify.json") || exit_code=$?
+          done
+          if [[ $exit_code -ne 0 ]]; then
+            echo "::error:: Please ensure verify phase is aligned on VIB pipeline files"
```
Comment (FraPazGal) [inline .github/workflows/vib-files-validator.yml:38]: I would clarify a bit more the comment so external contributors can find solve the issue easily:

```suggestion
            echo "::error:: Please ensure the VIB's verify phase is aligned both in vib-verify and vib-publish pipeline files"
```

Comment (fmulero) [inline .github/workflows/vib-files-validator.yml:38]: Could be great also if you could warn directly in one (or both) of the files:
```
echo "::error file=${dir}/vib-publish.json ::Please ensure verify phase is aligned on VIB pipeline files"
```

Comment (juan131) [reply .github/workflows/vib-files-validator.yml]: This mean we'll have to print an error message per discrepancy... wouldn't that be too verbose?

File: .github/workflows/vib-files-validator.yml (hunk 1 above)
Comment (juan131) [reply .github/workflows/vib-files-validator.yml]: done at https://github.com/bitnami/charts/pull/17808/commits/15a803bacaa1a8d7ca22665fbc3cfa00c8933500

File: .github/workflows/vib-files-validator.yml (hunk 3 above)
Comment (fmulero) [reply .github/workflows/vib-files-validator.yml]: As user if I change all vib files, for example to add a new linter or a new check, I'd like to know the file or the pair of files with the error. I think that is easy to add.

**Review by joancafom — APPROVED** [review]
LGTM! Thanks for your changes 🚀

**Pull request merged**